### PR TITLE
add rounding calibration function

### PR DESCRIPTION
**Pull request by sampocs**
Closes: #XXX

## Author's Checklist

I have...

- [ ] Run and PASSED locally all GAIA integration tests
- [ ] If the change is contentful, I either:
    - [ ] Added a new unit test OR 
    - [ ] Added test cases to existing unit tests
- [ ] OR this change is a trivial rework / code cleanup without any test coverage

If skipped any of the tests above, explain.

## Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] manually tested (if applicable)
- [ ] confirmed the author wrote unit tests for new logic
- [ ] reviewed documentation exists and is accurate

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes? 
  - [ ] Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`?
  - [ ] This pull request updates existing proto field values (and require a backend and frontend migration)? 
  - [ ] Does this pull request change existing proto field names (and require a frontend migration)?
  How is the feature or change documented? 
      - [ ] not applicable
      - [ ] jira ticket `XXX` 
      - [ ] specification (`x/<module>/spec/`) 
      - [ ] README.md 
      - [ ] not documented

### CODE REVIEW

File: x/stakeibc/client/cli/tx_calibrate_delegation.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	"github.com/Stride-Labs/stride/v14/x/stakeibc/types"
+)
+
+func CmdCalibrateDelegation() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "calibrate-delegation [chainid] [valoper]",
+		Short: "Broadcast message calibrate-delegation",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			argHostdenom := args[0]
```
Comment (asalzmann) [inline x/stakeibc/client/cli/tx_calibrate_delegation.go:18]:
```suggestion
			argChainId := args[0]
```

File: x/stakeibc/types/message_calibrate_delegation.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+func (msg *MsgCalibrateDelegation) ValidateBasic() error {
+	_, err := sdk.AccAddressFromBech32(msg.Creator)
+	if err != nil {
+		return errorsmod.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
+	}
+	if err := utils.ValidateAdminAddress(msg.Creator); err != nil {
+		return err
+	}
+
+	// basic checks on host denom
```
Comment (asalzmann) [inline x/stakeibc/types/message_calibrate_delegation.go:55]: comments seem off

File: x/stakeibc/client/cli/tx_calibrate_delegation.go (hunk 1 above)
Comment (sampocs) [reply x/stakeibc/client/cli/tx_calibrate_delegation.go]: done

File: x/stakeibc/types/message_calibrate_delegation.go (hunk 2 above)
Comment (sampocs) [reply x/stakeibc/types/message_calibrate_delegation.go]: they were just stale from the copy paste - removed

**Review by asalzmann — APPROVED** [review]

**Pull request merged**